src/logtab.cpp

```diff
@@ -171,22 +171,34 @@ void LogTab::keyPressEvent(QKeyEvent *event)
         }
         UpdateStatusBar();
         break;
-    case Qt::Key_C:
-        if ((QApplication::keyboardModifiers() & Qt::ControlModifier) &&
+    default:
+        // Check keys with modifiers together and ensure regular key events get propagated.
+        if ((event->key() == Qt::Key_C) &&
+            (QApplication::keyboardModifiers() & Qt::ControlModifier) &&
             (rowCount > 0))
         {
             CopyItemDetails(idxList);
         }
-        break;
-    case Qt::Key_D:
-        if ((QApplication::keyboardModifiers() & Qt::ControlModifier) &&
-            (rowCount == 2))
+        else if ((event->key() == Qt::Key_D) &&
+                 (QApplication::keyboardModifiers() & Qt::ControlModifier) &&
+                 (rowCount == 2))
         {
             RowDiffEvents();
         }
-        break;
-    default:
-        QWidget::keyPressEvent(event);
+        else if ((event->key() == Qt::Key_I) &&
```

Comment:
It's intended to be a "hidden" debugging/diagnostic info shortcut.  Currently it shows the file path of the tab, and for directory live capture - the list of files being monitored.  You can check this out from the internal tlv.win.latest file share.